minidump/src/minidump.rs

```diff
@@ -912,26 +912,31 @@ impl Module for MinidumpModule {
 
     fn code_identifier(&self) -> CodeId {
         match self.codeview_info {
-            Some(CodeView::Elf(ref raw)) => CodeId::from_binary(&raw.build_id),
-            Some(CodeView::Pdb70(ref raw)) if raw.age == 0 => {
-                // This is a macOS minidump. It looks like the age is typically
-                // >0 for Windows minidumps, and Breakpad sets it to 0 on macOS.
-                // We're relying on this observation in order to distinguish
-                // between these two platforms since macOS uses PDB70 instead of
-                // its own dedicated format.
-                // See the following linked comment in a Windows PDB writer:
-                // https://github.com/microsoft/microsoft-pdb/blob/082c5290e5aff028ae84e43affa8be717aa7af73/PDB/dbi/pdb.cpp#L602-L606
-                // TODO: Figure out a way to get access to `MinidumpSystemInfo`
-                // here instead of relying on `age`.
-                CodeId::new(format!("{:#}", raw.signature))
-            }
-            _ => {
-                // TODO: Breakpad stubs this out on non-Windows.
-                CodeId::new(format!(
-                    "{0:08X}{1:x}",
-                    self.raw.time_date_stamp, self.raw.size_of_image
-                ))
```

Comment:
Sorry for the massive diff here. I realized that the order of the match branches didn't... match the order used in other functions (e.g. `debug_identifier`, `debug_file`, etc) so I shuffled these around so they followed the same ordering as the rest of the file.

Reply:
You could split your change and the re-ordering of the matches in two separate patches. It helps with review and it makes the intention clearer when looking at the history.

Reply:
Thanks for the suggestion! I'm used to the aggressive squashing in sentry's repositories hence the single commit in the original branch; I've broken the reordering out so it's easier to follow what's going on based on the history.

This would mean that ideally this isn't squashed into the trunk if/once it is merged though. 🙏